docs/source/install.rst

```diff
@@ -121,6 +121,8 @@ To create initial data for the most used applications, run::
 
     $ ./manage.py create_initial_data
 
+See :doc: `../../cms/management/commands/commands` for the command options to specify while creating initial data.
```

Comment:
Wouldn't this link to the whole document? I'd prefer linking to the `create_initial_command` directly to make it more future proof:

```rst
.. In commands.rst, add this:

.. _command-create-initial-data:

create_initial_data
-------------------

[...]
```

And then use it in `install.rst`:

```rst
See :ref:`command-create-initial-data` for the command options [...]
```